query-engine/query-engine-node-api/Cargo.toml

```diff
@@ -23,8 +23,8 @@ psl.workspace = true
 sql-connector = { path = "../connectors/sql-query-connector", package = "sql-query-connector" }
 prisma-models = { path = "../prisma-models" }
 
-napi = { version = "2", default-features = false, features = ["napi8", "tokio_rt", "serde-json"] }
-napi-derive = "2"
+napi = { version = "2.9.0", default-features = false, features = ["napi8", "tokio_rt", "serde-json"] }
```

Comment:
This form of dependency means "2.9.0 or later", it will not prevent `cargo update` from upgrading to anything below 3.0. But it's possible to pin by writing `"=2.9.0"`. I _think_ it's the same rules as npm.

Reply:
Fixed